internal/cmd/ske/credentials/complete-rotation/complete_rotation.go

```diff
@@ -0,0 +1,125 @@
+			"Completes the rotation of the credentials associated to a STACKIT Kubernetes Engine (SKE) cluster.",
+			"This is step 2 of a 2-step process to rotate all SKE cluster credentials. Tasks accomplished in this phase include:",
+			"  - The old certification authority will be dropped from the package.",
+			"  - The old signing key for the service account will be dropped from the bundle.",
+			"To ensure continued access to the Kubernetes cluster, please update your kubeconfig with the new credentials:",
+			"  $ stackit ske kubeconfig create my-cluster",
+			"If you haven't, please start the process by running:",
+			"  $ stackit ske credentials start-rotation my-cluster",
+			"For more information, visit: https://docs.stackit.cloud/stackit/en/how-to-rotate-ske-credentials-200016334.html",
+		),
+		Args: args.SingleArg(clusterNameArg, nil),
+		Example: examples.Build(
+			examples.NewExample(
+				`Complete the rotation of the credentials associated to the SKE cluster with name "my-cluster"`,
+				"$ stackit ske credentials complete-rotation my-cluster",
+			),
+			examples.NewExample(
+				`Flow of the 2-step process to rotate all SKE cluster credentials, including generating a new kubeconfig file`,
+				"$ stackit ske credentials start-rotation my-cluster",
+				"$ stackit ske kubeconfig create my-cluster",
+				"$ stackit ske credentials complete-rotation my-cluster",
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd, args)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			if !model.AssumeYes {
+				prompt := fmt.Sprintf("Are you sure you want to complete the rotation of the credentials for SKE cluster %q?", model.ClusterName)
+				err = p.PromptForConfirmation(prompt)
+				if err != nil {
+					return err
+				}
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			_, err = req.Execute()
+			if err != nil {
+				return fmt.Errorf("complete rotation of SKE credentials: %w", err)
+			}
+
+			// Wait for async operation, if async mode not enabled
+			if !model.Async {
+				s := spinner.New(p)
+				s.Start("Completing credentials rotation")
+				_, err = wait.CompleteCredentialsRotationWaitHandler(ctx, apiClient, model.ProjectId, model.ClusterName).WaitWithContext(ctx)
+				if err != nil {
+					return fmt.Errorf("wait for completing SKE credentials rotation %w", err)
+				}
+				s.Stop()
+			}
+
+			operationState := "Rotation of credentials is completed"
+			if model.Async {
+				operationState = "Triggered completion of credentials rotation"
+			}
+			p.Info("%s for cluster %q\n", operationState, model.ClusterName)
+			p.Info("Consider updating your kubeconfig with the new credentials, create a new kubeconfig by running:\n  $ stackit ske kubeconfig create %s\n", model.ClusterName)
```

Comment:
Not sure if this one is `Info` or `Warning`

Reply:
But probably we can leave it as `Info`

Reply:
That is indeed a warning, will fix in next story